bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -37,48 +24,48 @@ The variables you will need to get to setup the binding are:
 * `<PASSWORD>` The password attached to the above username.
 
 
-### 2. Bridge and Things Configuration
+The binding has the following configuration options:
```

Comment:
I would like to renew my concerns originally expressed here: https://github.com/openhab/openhab-addons/pull/9486/files#r584111632

Configuring the account on binding level makes it impossible to use two accounts. It's far better to do that at bridge level.

Reply:
That's a very valid and good point!

Previous configuration for two weather stations with one additional indoor module each:

```
Bridge netatmo:netatmoapi:home "Netatmo API" [clientId="", clientSecret="", username = "", password = "", readStation=true, readHealthyHomeCoach=false, readThermostat=false, readWelcome=false, readPresence=false] {
    Thing NAMain    indoor_familyroom [ id="xx:xx:xx:xx:xx:01" ]
    Thing NAModule4 indoor_bedroom    [ id="xx:xx:xx:xx:xx:02", parentId="xx:xx:xx:xx:xx:01" ]

    Thing NAMain    indoor_office     [ id="xx:xx:xx:xx:xx:11 ]
    Thing NAModule4 indoor_alva       [ id="xx:xx:xx:xx:xx:12", parentId="xx:xx:xx:xx:xx:11" ]
}
```

I slightly disliked this approach since the bridge was the API and the physical bridge (main weather station module) was not defined as a bridge. So for example, when main module was offline, logically additional modules wouldn't automatically be taken offline by openHAB, even though their physical bridge was offline.

New configuration:

```
Bridge netatmo:NAMain:indoor_familyroom [ id="xx:xx:xx:xx:xx:01" ] {
    NAModule4 indoor_bedroom [ id="xx:xx:xx:xx:xx:02" ]
}

Bridge netatmo:NAMain:indoor_office [ id="xx:xx:xx:xx:xx:11" ] {
    NAModule4 indoor_alva [ id="xx:xx:xx:xx:xx:12" ]
}
```

Now physical bridges are defined as bridges, but you can't have multiple accounts, as you have pointed out. Maybe something like this would be acceptable:

```
Bridge netatmo:NAMain:indoor_familyroom [ id="xx:xx:xx:xx:xx:01", clientId="", clientSecret="", username = "", password = "" ] {
    NAModule4 indoor_bedroom [ id="xx:xx:xx:xx:xx:02" ]
}

Bridge netatmo:NAMain:indoor_office [ id="xx:xx:xx:xx:xx:11", clientId="", clientSecret="", username = "", password = "" ] {
    NAModule4 indoor_alva [ id="xx:xx:xx:xx:xx:12" ]
}
```

Now the API parameters are duplicated in my setup having two weather stations, but maybe that's the price to pay for being able to use separate accounts?

Reply:
Perhaps configuration at binding and bridge level could be combined, so the API parameters could optionally be configured at binding level, but optionally overridden at bridge level?

Reply:
>It's far better to do that at bridge level.

I agree

Reply:
Okay, now read through first version of the PR and found this comment: https://github.com/openhab/openhab-addons/pull/9486#issuecomment-765621918

So if it's possible to have nested bridges like that, that would probably be better.

Reply:
Yes it is possible to nest bridges as much as wanted. I did not want to do it prior to yesterday commit because everything in the binding was a bridge (explained in some earliers comments).
I'm still not convinced that anyone will be in a configuration to have multiple Netatmo accounts, as you can link as many systems as you want at Netatmo level, but well...with this last evolution it is far easier so I'm going to get back to my bench and define an API bridge.

Reply:
@J-N-K : your request is now taken in account. You can have multiple bridges.